LPD-44371 Build Service

sandrodw3 opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

dgarciasarai commented:
Sandro, I close this pull request because @andresfulla is going to add more code. He will resend it, thanks.

JhossephG committed:
LPD-44524 Adding original value from custom fields to system objects payload json

JhossephG committed:
LPD-44524 Return new field added in json when oldValue is true

JhossephG committed:
LPD-44524 Add test

ctang3 committed:
LPD-44987 Add finder to create a unique index that includes ctCollectionId

ctang3 committed:
LPD-44987 Add constraint resolver for AssetEntryAssetCategoryRel

ctang3 committed:
LPD-44987 buildLang

ctang3 committed:
LPD-44987 buildService

Commit:
LRSD-7806 Add 'Go to DEV' option to the Liferay SaaS Activation tab

pedro-oliveira446 committed:
LPD-44371 Create new public method to deploy all object definitions

pedro-oliveira446 committed:
LPD-44371 Integration Test

carolmariaabb committed:
LPD-45012 Show related Object Fields in the expression builder sidebar for Formula fields

carolmariaabb committed:
LPD-45012 Add integration test

carolmariaabb committed:
LPD-45012 Add query expressions for related object field columns when evaluating the DDMExpression for formula fields

carolmariaabb committed:
LPD-45012 Add integration test

pedro-oliveira446 committed:
LPD-44371 Build Service

kevenleone committed:
LPD-45313 Throw exception if status code is either 4xx or 5xx

ericyanLr committed:
LPD-45236 Add test to expose issue

Assert the loginDate and lastLoginDate is stored with the correct
date and timestamp when initially updated or when updated using the
BatchProcessor.

ericyanLr committed:
LPD-45236 Use setTimestamp to store both the date and respective time

Using setDate can properly store the date, but not the respective
time. It instead normalizes the time to 00:00:00.

ling-alan-huang committed:
LPD-44271 Auto SF

ling-alan-huang committed:
LPD-44271 Auto SF

ling-alan-huang committed:
LPD-44271 Auto SF

oliv-yu committed:
LPD-45174 portal-search-web: Close modal before saving blueprint/element

Commit:
LPD-44208 Jakarta cache

tinatian committed:
LPD-45236 Simplify test logic

michaelhashimoto committed:
LRCI-5161 Remove mirrors hostname references

michaelhashimoto committed:
LRCI-5161 Add method to set patch shas for hotfix tests

ling-alan-huang committed:
LPD-43790 Remove duplicate descriptions

ling-alan-huang committed:
LPD-43790 gw buildLang

ling-alan-huang committed:
LPD-43790 Change language keys

ling-alan-huang committed:
LPD-43790 Auto SF

ling-alan-huang committed:
LPD-43790 Remove duplicate descriptions

ling-alan-huang committed:
LPD-43790 gw buildLang

ling-alan-huang committed:
LPD-43790 Change language keys

ling-alan-huang committed:
LPD-43790 Auto SF

ling-alan-huang committed:
LPD-43477 SF

ling-alan-huang committed:
LPD-43477 SF

ling-alan-huang committed:
LPD-43477 SF

petershin committed:
LPD-44208 Output directory

petershin committed:
LPD-44208 Clean cache

petershin committed:
LPD-44208 Source formatting

brianchandotcom committed:
LPD-34518 prep next

brianchandotcom committed:
LPD-34518 prep next

brianchandotcom committed:
LPD-34518 Just master

kenjiheigel committed:
LRCI-5067 Ignore particular email error

brianchandotcom committed:
LRCI-5161 prep next

brianchandotcom committed:
LPD-44208 prep next

brianchandotcom committed:
LPD-45012 Rename

brianchandotcom committed:
LPD-44524 Do we test for custom?

brianchandotcom committed:
LPD-44987 Wordsmith

brianchandotcom committed:
LPD-44987 Regen

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
ci:test:playwright

liferay-continuous-integration commented:
<html><h3>:x: ci:test:playwright - 0 out of 1 jobs passed in 7 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/61b58512ab7f586f7ca6824d9a013acf90aafc3f">61b58512ab7f586f7ca6824d9a013acf90aafc3f</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/83a242f48ab4b2086965005a590a95e42e9c0a13">83a242f48ab4b2086965005a590a95e42e9c0a13</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/103654631">EE Development Acceptance (master) - 1117 - 2025-01-04[12:36:18]</a></p><details><summary><strong>ci:test:playwright - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/10852/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/10852/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/10852/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><pre>Started by remote host 172.16.168.126
Running as SYSTEM
[EnvInject] - Loading node environment variables.
Building remotely on test-1-29-79 (slave-1) in workspace /opt/dev/projects/github/liferay-jenkins-ee
[liferay-jenkins-ee] \$ /bin/bash -e -x /tmp/jenkins17144420416276736084.sh
+ echo 'Initialize Jenkins slave node.'
Initialize Jenkins slave node.
+ cd /opt/dev/projects/github/liferay-jenkins-ee
+ curl -o ./_common.sh http://mirrors-no-cache.lax.liferay.com/github.com/liferay/liferay-jenkins-ee/_common.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 19380  100 19380    0     0  1948k      0 --:--:-- --:--:-- --:--:-- 2102k
+ source ./_common.sh
++ MAX_THREADS=1
++ [[ test-1-29-79.lax.liferay.com =~ cloud-.* ]]
++ [[ test-1-29-79 slave-1 =~ .*win.* ]]
++ ROOT_DIR=
++ GIT_REPOSITORIES_ROOT_DIR=/opt/dev/projects/github
++ JENKINS_REPO_PATH=/opt/dev/projects/github/liferay-jenkins-ee
+++ cat /opt/dev/projects/github/liferay-jenkins-ee/git-directories.json
++ GIT_DIRECTORIES_JSON_CONTENT='[
  \{
    "branch": "master",
    "name": "alloy-ui",
    "repository": "alloy-ui",
    "type": "mounted"
  \},
  \{
    "branch": "7.0.x",
    "name": "com-liferay-osb-asah-private",
    "repository": "com-liferay-osb-asah-private",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-binaries-cache-2017",
    "repository": "liferay-binaries-cache-2017",
    "type": "shared-mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-binaries-cache-2020",
    "repository": "liferay-binaries-cache-2020",
    "type": "shared-mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-blade-samples",
    "repository": "liferay-blade-samples",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-docker",
    "repository": "liferay-docker",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-alloy",
    "repository": "liferay-faces-alloy",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-bridge-impl",
    "repository": "liferay-faces-bridge-impl",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-portal",
    "repository": "liferay-faces-portal",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-showcase",
    "repository": "liferay-faces-showcase",
    "type": "mounted"
  \},
  \{
    "branch": "master",</pre></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

sandrodw3 commented:
ci:test:playwright

liferay-continuous-integration commented:
<html><h3>:x: ci:test:playwright - 0 out of 1 jobs passed in 6 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/61b58512ab7f586f7ca6824d9a013acf90aafc3f">61b58512ab7f586f7ca6824d9a013acf90aafc3f</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/83a242f48ab4b2086965005a590a95e42e9c0a13">83a242f48ab4b2086965005a590a95e42e9c0a13</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/103654631">EE Development Acceptance (master) - 1117 - 2025-01-04[12:36:18]</a></p><details><summary><strong>ci:test:playwright - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/12099/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/12099/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/12099/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><pre>Started by remote host 172.16.168.126
Running as SYSTEM
[EnvInject] - Loading node environment variables.
Building remotely on test-1-28-37 (slave-1) in workspace /opt/dev/projects/github/liferay-jenkins-ee
[liferay-jenkins-ee] \$ /bin/bash -e -x /tmp/jenkins1588605694474777495.sh
+ echo 'Initialize Jenkins slave node.'
Initialize Jenkins slave node.
+ cd /opt/dev/projects/github/liferay-jenkins-ee
+ curl -o ./_common.sh http://mirrors-no-cache.lax.liferay.com/github.com/liferay/liferay-jenkins-ee/_common.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 19380  100 19380    0     0  1877k      0 --:--:-- --:--:-- --:--:-- 1892k
+ source ./_common.sh
++ MAX_THREADS=1
++ [[ test-1-28-37.lax.liferay.com =~ cloud-.* ]]
++ [[ test-1-28-37 slave-1 =~ .*win.* ]]
++ ROOT_DIR=
++ GIT_REPOSITORIES_ROOT_DIR=/opt/dev/projects/github
++ JENKINS_REPO_PATH=/opt/dev/projects/github/liferay-jenkins-ee
+++ cat /opt/dev/projects/github/liferay-jenkins-ee/git-directories.json
++ GIT_DIRECTORIES_JSON_CONTENT='[
  \{
    "branch": "master",
    "name": "alloy-ui",
    "repository": "alloy-ui",
    "type": "mounted"
  \},
  \{
    "branch": "7.0.x",
    "name": "com-liferay-osb-asah-private",
    "repository": "com-liferay-osb-asah-private",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-binaries-cache-2017",
    "repository": "liferay-binaries-cache-2017",
    "type": "shared-mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-binaries-cache-2020",
    "repository": "liferay-binaries-cache-2020",
    "type": "shared-mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-blade-samples",
    "repository": "liferay-blade-samples",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-docker",
    "repository": "liferay-docker",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-alloy",
    "repository": "liferay-faces-alloy",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-bridge-impl",
    "repository": "liferay-faces-bridge-impl",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-portal",
    "repository": "liferay-faces-portal",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-showcase",
    "repository": "liferay-faces-showcase",
    "type": "mounted"
  \},
  \{
    "branch": "master",</pre></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>
